jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/viewmodel/LanguageSelectionViewModel.kt

```diff
@@ -47,7 +47,7 @@ class LanguageSelectionViewModel(items: ObservableList<Language>) : ViewModel()
     init {
         selectedRegions.onChange {
             regionPredicate = if (it.list.isEmpty()) {
-                Predicate { false }
+                Predicate { true }
```

Comment:
Explanation: displays all result when no box is checked.

Previously:
![image](https://user-images.githubusercontent.com/34975907/145089009-fc1be3eb-63c0-4e1f-8cb4-86a42ffc4d17.png)